client/app/phpstan-baseline.neon

```diff
@@ -1922,7 +1922,7 @@ parameters:
 
 		-
 			message: "#^Cannot call method getData\\(\\) on Symfony\\\\Component\\\\Form\\\\FormInterface\\|null\\.$#"
-			count: 2
+			count: 3
```

Comment:
The fix for this is to cast the FormInterface object to `Symfony\Component\Form\Form`, so that it has a `getData()` method available. I'll let this go for now, but rather than add more phpstan errors to the baseline we should fix them as we go.